src/main/kotlin/step3/model/TrackLine.kt

```diff
@@ -0,0 +1,12 @@
+package step3.model
+
+class TrackLine(val car: Car) {
+
+    var position: Int = 0
```

Comment:
`var`로 선언된 필드인데 public이다보니 외부에서 이 필드를 직접 조작할 수도 있겠네요. 아래처럼 setter는 private으로 선언하면 어떨까요?

```suggestion
    var position: Int = 0
        private set
```